README.md

```diff
@@ -1 +1,217 @@
+# or
+yarn add @pagopa/io-react-native-cie
+```
+
+## Example App
+
+A fully working example app demonstrating how to use this package is available in the [example](./example) directory.
+
+To run the example app, follow the instructions in [example/README.md](./example/README.md).
+
+## Usage
+
+The package is split into two modules:
+
+- [CieUtils](/src/utils): Provides functions to check the NFC status of the device.
+- [CieManager](/src/manager): Provides CIE read and authentication capabilities.
+
+### Check NFC Status
+
+**Note:** These methods are applicable only for Android devices, as iOS devices always have NFC available.
+
+```typescript
+import { CieUtils } from '@pagopa/io-react-native-cie';
+
+// Check if the device has NFC
+await CieUtils.hasNfcFeature();
+// Check if NFC is enabled
+await CieUtils.isNfcEnabled();
+// Convenient method to check if CIE authentication is supported
+await CieUtils.isCieAuthenticationSupported();
+```
+
+### Reading CIE Data
+
+#### Reading Attributes
+
+Read CIE attributes (card type and base64-encoded data):
+
+```typescript
+import { CieManager } from '@pagopa/io-react-native-cie';
+
+// Start reading with optional timeout (Android only)
+CieManager.startReadingAttributes(timeout?: number)
+  .then(() => console.log('Reading started'))
+  .catch(error => console.error('Error:', error));
+```
+
+#### Authentication
+
+Start the CIE authentication process:
+
+```typescript
+CieManager.startReading(
+  pin: string,           // CIE card PIN
+  authenticationUrl: string,  // Service URL
+  timeout?: number       // Optional timeout (Android only)
+)
+  .then(() => console.log('Authentication started'))
+  .catch(error => console.error('Error:', error));
+```
+
+### Event System
+
+The library uses an event-driven approach for NFC operations. Events are emitted to notify your application about progress, success, or failure. Each listener method returns a cleanup function that should be called when the operation is complete or when your component unmounts.
+
+#### Available Event Listeners
+
+| Listener Type                  | Description                |
+| ------------------------------ | -------------------------- |
+| `addEventListener`             | General NFC events         |
+| `addErrorListener`             | NFC error events           |
+| `addSuccessListener`           | Authentication success     |
+| `addAttributesSuccessListener` | Successful attribute reads |
+
+#### Example Usage
+
+```typescript
+import { CieManager } from '@pagopa/io-react-native-cie';
+import { useEffect } from 'react';
+
+// ...
+
+useEffect(() => {
+  // Register all event listeners
+  const cleanup = [
+    CieManager.addEventListener((event) => {
+      console.info('NFC Event', event);
+    }),
+    CieManager.addErrorListener((error) => {
+      console.error('NFC Error', error);
+    }),
+    CieManager.addSuccessListener((url) => {
+      console.log('Auth url:', url);
+    }),
```

Comment:
Would it make sense to include a table that maps the current SDK events (e.g., [these](https://github.com/pagopa/io-cie-sdk?tab=readme-ov-file#possibili-eventi)) to the new implementation?
It could help make the migration path clearer.

Reply:
Yes absolutely. Added here: https://github.com/pagopa/io-react-native-cie/pull/1/commits/8893643908bfd1f098464a26ffc79658ac812170

Thank you!

Reply:
I've added a [migration guide](https://github.com/pagopa/io-react-native-cie/blob/SIW-2265-first-implementation/docs/migrate-from-io-cie-sdk.md) that explain how to replace current `react-native-cie` implementation with the new `io-react-native-cie` methods. Let me know what you think!